app/views/shared/_size_and_zone_filters.html.erb

```diff
@@ -0,0 +1,86 @@
+<div class="dropdown show mb-8px mr-8px">
+  <a
+    id="dropdownMenuLink"
+    class="btn btn-sm btn-outline btn-secondary dropdown-toggle"
+    href="#"
+    role="button"
+    data-toggle="dropdown"
+    data-display="static"
+    aria-haspopup="true"
+    aria-expanded="false"
+  >
+    <% if @selected_zones.size == 1 %>
+      <%= @selected_zones.first.capitalize %>
+    <% else %>
+      All blocks
+    <% end %>
+  </a>
+  <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
```

Comment:
The zones dropdown seems to `overflow: scroll` on mobile but it's getting huge on desktop. Shall we set a  `max-height` on desktop as well?
![image](https://user-images.githubusercontent.com/16774200/97534639-5dae3a00-19e0-11eb-8b69-129565111cf5.png)

Reply:
Great point! Setting `max-height` on desktop 👍